test/parallel/test-readline-async-iterators-backpressure.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Readable } = require('stream');
+const readline = require('readline');
+
+const CONTENT = 'content';
+const TOTAL_LINES = 18;
+
+(async () => {
+  const readable = new Readable({ read() {} });
```

Comment:
Looking at this now (to solve another bug) I think this backpressure behaviour is confusing since other consumers can listen to `line` on the stream and it's surprising we `pause()` it for them.